docs/core/install/linux-ubuntu.md

```diff
@@ -38,30 +37,44 @@ The following table is a list of currently supported .NET releases and the versi
 
 ## 22.04
 
-[!INCLUDE [linux-prep-intro-apt](includes/linux-prep-intro-apt.md)]
+> Warning: If you've previously installed .NET 6 from `packages.microsoft.com`, see the [Advisory on installing .NET 6 on Ubuntu 22.04].
+
+.NET 6 is included in Ubuntu 22.04. You can install it with the following commands:
+
+For the .NET SDK:
 
 ```bash
-wget https://packages.microsoft.com/config/ubuntu/22.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
-sudo dpkg -i packages-microsoft-prod.deb
-rm packages-microsoft-prod.deb
+sudo apt install -y dotnet6
 ```
 
-[!INCLUDE [linux-apt-install-60](includes/linux-install-60-apt.md)]
+For ASP.NET Core
 
-Note: [Ubuntu 22.04 includes OpenSSL 3](https://discourse.ubuntu.com/t/openssl-3-0-transition-plans/24453) as the baseline version. [.NET 6 supports OpenSSL 3](https://devblogs.microsoft.com/dotnet/announcing-net-6/#security) while earlier .NET versions do not. Microsoft does not test or support using OpenSSL 1.x on Ubuntu 22.04.
+```bash
+sudo apt install -y apt install -y aspnetcore-runtime-6.0
+```
+
+For .NET Runtime
+
+```bash
+sudo apt install -y apt install -y dotnet-runtime-6.0
+```
 
-## 21.10
+## 22.04 -- packages.microsoft.com
+
+> Warning: .NET 6 is included in Ubuntu 22.04. See the [Advisory on installing .NET 6 on Ubuntu 22.04] if you want to use .NET packages from `packages.microsoft.com`.
 
 [!INCLUDE [linux-prep-intro-apt](includes/linux-prep-intro-apt.md)]
 
 ```bash
-wget https://packages.microsoft.com/config/ubuntu/21.10/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
+wget https://packages.microsoft.com/config/ubuntu/22.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
 sudo dpkg -i packages-microsoft-prod.deb
 rm packages-microsoft-prod.deb
 ```
 
 [!INCLUDE [linux-apt-install-60](includes/linux-install-60-apt.md)]
 
+Note: [Ubuntu 22.04 includes OpenSSL 3](https://discourse.ubuntu.com/t/openssl-3-0-transition-plans/24453) as the baseline version. [.NET 6 supports OpenSSL 3](https://devblogs.microsoft.com/dotnet/announcing-net-6/#security) while earlier .NET versions do not. Microsoft does not test or support using OpenSSL 1.x on Ubuntu 22.04.
```

Comment:
Note syntax here, and use conjunctions for **do not** and **does not**. We also don't use inline links when ending with information, use the **For more information, see ...** syntax:

```suggestion
> [!NOTE]
> [Ubuntu 22.04 includes OpenSSL 3](https://discourse.ubuntu.com/t/openssl-3-0-transition-plans/24453) as the baseline version. .NET 6 supports OpenSSL 3 while earlier .NET versions don't. Microsoft doesn't test or support using OpenSSL 1.x on Ubuntu 22.04. For more information, see [.NET 6 Security Improvements](https://devblogs.microsoft.com/dotnet/announcing-net-6/#security).
```